cpp/jwt/jwtWrapper.cpp

```diff
@@ -130,7 +130,20 @@ const char * CJWT_serialize(CJWT cjwt_obj, EVP_PKEY *key)
 int CJWT_verify(CJWT cjwt_obj, EVP_PKEY *key)
 {
     IJwt *ijwt_obj = static_cast<IJwt *>(cjwt_obj);
-    if ((ijwt_obj->verify(key, false)))
+    if ((ijwt_obj->verify(key)))
+    {
+        return 1;
+    }
+    else
+    {
+        return 0;
+    }
+}
+
+int CJWT_verify_token(CJWT cjwt_obj, EVP_PKEY *key, const char *token)
```

Comment:
IMO it would probably be better if we can hide this `token` from the interface and make it a member of the JWTobject. Therefore the JWT object could be self contained once established without requiring the user to pass in the token.